### PR TITLE
another controllers.png icon

### PR DESCRIPTION
**Pull request by feiss**
I think this icon is more readable and clear. Old one was quite confusing..

### CODE REVIEW

File: themes/aframe/layout/partials/examples/sidebar.ejs [hunk 1]
```diff
@@ -35,7 +35,6 @@
                  class="subnav-link<%= className %>">
                 <span class="sidebar__link__text"><%- item.title %></span>
                 <% if (item.supports.controllers) { %>
-                  <!-- Image modified from https://thenounproject.com/term/oculus-controller/197923/ by David Swanson. https://creativecommons.org/licenses/by/3.0/us/ -->
```
Comment (cvan) [inline themes/aframe/layout/partials/examples/sidebar.ejs]: is there a source for this new controller or did you make it?

**Comment by cvan** [comment]
was `src/images/awoa/first-person-sh*tter.jpg` deleted intentionally?

**Comment by feiss** [comment]
nope! not at all. The name with the * causes a lot of pain to git and filesystems.. I would fix it, but maybe you guys wanted to take a look to it

File: themes/aframe/layout/partials/examples/sidebar.ejs (hunk 1 above)
Comment (feiss) [inline themes/aframe/layout/partials/examples/sidebar.ejs]: I made it!

**Comment by ngokevin** [comment]
Thanks! The old one was just a placeholder I could find for free until you created something better :)

**Comment by cvan** [comment]
> nope! not at all. The name with the * causes a lot of pain to git and filesystems.. I would fix it, but maybe you guys wanted to take a look to it

can you just remove the `*` and fix it in `src/_posts/awoa-08.md`?

File: themes/aframe/layout/partials/examples/sidebar.ejs (hunk 1 above)
Comment (cvan) [inline themes/aframe/layout/partials/examples/sidebar.ejs]: nice work 👍

**Comment by cvan** [comment]
fixed rename in PR #413

**Pull request merged**